frontend/src/features/mapSideBar/SidebarContents/LandForm.tsx

```diff
@@ -401,15 +385,6 @@ const LandForm: React.FC<IParentLandForm> = (props: IParentLandForm) => {
       validation: props.disabled ? undefined : { schema: ValuationSchema, nameSpace: () => 'data' },
     },
   ];
-
-  if (!!props.initialValues?.buildings?.length) {
```

Comment:
I'm actually really happy we were able to take this out, because it makes this code a lot cleaner. Conditional steps are a pain.